thorlcr/slave/thslavemain.cpp

```diff
@@ -117,7 +117,7 @@ static bool RegisterSelf(SocketEndpoint &masterEp)
 {
     StringBuffer slfStr;
     StringBuffer masterStr;
-    LOG(MCdebugProgress, "registering %s - master %s",slfEp.getEndpointHostText(slfStr).str(),masterEp.getEndpointHostText(masterStr).str());
+    PROGLOG("registering %s - master %s",slfEp.getEndpointHostText(slfStr).str(),masterEp.getEndpointHostText(masterStr).str());
```

Comment:
Is this something that should go to the user?  Should really be named UPROGLOG to make it clear..

Reply:
I'm going to introduce UPROGLOG (we have same for UERRLOG vs ERRLOG already) and rename all PROGLOG's to UPROGLOG in Thor.

I think it is debatable whether many current user progress logs should be user or programmer (or perhaps a different audience). If an advanced user wants to see the engine making progress, then some lines like these are useful - but for average ECL user, very few are meaningful..

Reply:
It would be worth us clearly defining which messages should go to which targets.
It would also make more sense if D was used as the prefixed for debug/programmers rather than I.  That would make the functions more consistent.  I will take a look at what is there already.